### ISSUE TITLE
CIGAR and sequence length are inconsistent

### ISSUE THREAD

**Issue by Unaimend** [CSB5/OPERA-MS#51]
So I got this error while assembling a dataset, could some tell me how to investigate the source of this error?

[main] Version: 0.7.10-r789
[main] CMD: /scratch/SlurmTMP/sukem127.571191/OPERA-MS//tools_opera_ms/bwa index -p /scratch/SlurmTMP/sukem127.571191/RESULTS//intermediate_files/coverage_estimation/final.contigs.fa /scratch/SlurmTMP/sukem127.571191/RESULTS/intermediate_files/megahit_assembly/final.contigs.fa
[main] Real time: 698.433 sec; CPU: 687.584 sec
[M::main_mem] read 1333334 sequences (200000100 bp)...
[M::mem_process_seqs] Processed 1333334 reads in 246.059 CPU sec, 12.599 real sec
[samopen] SAM header is present: 189153 sequences.
Line 251087, sequence length 101 vs 150 from CIGAR
Parse error at line 251087: CIGAR and sequence length are inconsistent

**Comment by jsgounot** [comment]
Hi,

thanks for using OPERA-MS. I will need some materials to investigate this. Do you mind sharing the opera-ms command line which led to this result ? Additionally, could you run [this script](https://github.com/CSB5/OPERA-MS/blob/OPERA-MS-0.9.0/bin/get_log.sh) with your result folder as argument and send me the logs archive ?

Regards,
Jean-Sebastien

**Comment by Unaimend** [comment]
Thanks for your quick  reply
Here is the command line
perl OPERA-MS/OPERA-MS.pl \
    --short-read1 $_my_first_read \
    --short-read2 $_my_second_read \
    --long-read $_my_long_read \
    --no-ref-clustering \
    --num-processors 30 \
    --out-dir RESULTS 2> log.err

Regards,
Thomas Dost

**Comment by Unaimend** [comment]
[OPERA-MS-LOG.tar.gz](https://github.com/CSB5/OPERA-MS/files/5841095/OPERA-MS-LOG.tar.gz)

And here are the requested log files

**Comment by jsgounot** [comment]
Hi, sorry for the very long reply. Were you able to find a solution to this issue ?

**Comment by Unaimend** [comment]
No, I did not investigate this issue any further  since I was not able to generate "good" reads with PbSim. After switching to badreads I did not have any problems. But I will look into after finishing my Bachelor Thesis.
